testsuite/cluster-test/src/tx_emitter.rs

```diff
@@ -88,7 +88,7 @@ impl TxEmitter {
         }
     }
 
-    pub fn start_job(&mut self, req: EmitJobRequest) -> EmitJob {
+    pub fn start_job(&mut self, req: EmitJobRequest) -> failure::Result<EmitJob> {
```

Comment:
For consistency, can we update `fn stop_job` to return a Result as well?

Reply:
I disagree with consistency point - since `stop_job` can not fail currently, there is no point in returning Result there